src/Cli/dotnet/NugetPackageInstaller/PackageSourceLocation.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.Cli.NuGetPackageDownloader
+{
+    internal class PackageSourceLocation
+    {
+        public PackageSourceLocation(
+            FilePath? nugetConfig = null,
+            DirectoryPath? rootConfigDirectory = null, 
+            string[] overrideSourceFeeds = null)
+        {
+            NugetConfig = nugetConfig;
+            RootConfigDirectory = rootConfigDirectory;
+            ExpandLocalFeedAndAssign(overrideSourceFeeds);
+        }
+
+        private void ExpandLocalFeedAndAssign(string[] overrideSourceFeeds)
```

Comment:
Should this be named sourceFeedOverrides? overrideSourceFeeds implies more of a boolean action whereas the parameter seems to be supplying an array over feed overrides.